backend/src/Manager/Order/OrderManager.php

```diff
@@ -181,4 +182,21 @@ public function orderUpdateCaptainOrderCost(OrderUpdateCaptainOrderCostRequest $
 
         return $orderEntity;
     }
+    
+    public function updateCaptainArrived(OrderUpdateCaptainArrivedRequest $request): ?OrderEntity
+    {
+        $orderEntity = $this->orderRepository->find($request->getId());
+
+        if(! $orderEntity) {
+            return $orderEntity;
+        }
+               
+        $orderEntity = $this->autoMapping->mapToObject(OrderUpdateCaptainArrivedRequest::class, OrderEntity::class, $request, $orderEntity);
+        
+        $orderEntity->setDateCaptainArrived(new DateTime());
```

Comment:
Is this set the current date? Why not _new DateTime('Now')_?

Reply:
There is no difference